demonstrations/tutorial_tn_basics.metadata.json

```
@@ -0,0 +1,49 @@
+{
+    "title": "Tutorial: ABC of tensor networks",
+    "authors": [
+        {
+            "username": "emiliano"
+        },
+        {
+            "username": "ShiroRaven"
+        }
+    ],
+    "dateOfPublication": "2024-08-06T00:00:00+00:00",
+    "dateOfLastModification": "2024-08-06T00:00:00+00:00",
+    "categories": [
+        "Getting Started"
+    ],
+    "tags": [],
+    "previewImages": [
+        {
+            "type": "thumbnail",
+            "uri": "/_static/demo_thumbnails/regular_demo_thumbnails/thumbnail_tensor_network_quantum_circuits.png"
+        }
+    ],
+    "seoDescription": "An introductory tutorial to tensor networks.",
```

This SEO description is too short and lacks substance.

Do you have any thoughts on an improved on @alvaro-at-xanadu @EmilianoG-byte, and which keywords we should include ?
An idea below:

```suggestion
    "seoDescription": "Discover the fundamentals of tensor networks with this comprehensive demo. Learn how tensors extend vectors and matrices, explore their intuitive diagrams, and see how they connect to quantum circuits.",
```

---

How about something similar to yours emphasising its applicability to both beginners and advanced users:

```suggestion
    "seoDescription": "Discover the fundamentals of tensor networks with this comprehensive demo. Learn how tensors generalize vectors and matrices, explore their intuitive diagrams, and see how they connect to quantum circuits, with insights for both beginners and advanced readers."",
```

---

Looks good to me! :+1: